tutoraspects/templates/openedx-assets/assets/dashboards/Instructor_Dashboard.yaml

```diff
@@ -1,6 +1,6 @@
 _file_name: Instructor_Dashboard.yaml
 _roles:
-- {{ SUPERSET_ROLES_MAPPING.instructor }}
+- "{{ SUPERSET_ROLES_MAPPING.instructor }}"
 css: ''
```

Comment:
For the localization process to work now we need to wrap all curly braces in double quotes. This still needs testing on the translations_pull side once the existing translations are updated, but I think it will work.